softvis3d-frontend/package.json

```diff
@@ -64,6 +64,8 @@
     "css-loader": "^0.26.1",
     "enzyme": "^2.7.0",
     "extract-text-webpack-plugin": "^1.0.1",
+    "jsdom": "9.10.0",
```

Comment:
`jsdom` provides us a virtual browser dom (and with that a global window-object). This is required to `mount()` components instead of only `shallow()` renders. (See [here](https://github.com/stefanrinderle/softvis3d/pull/73/files#diff-8154fde546bf80f6f4e7b53163bffc46R32))

Without mounting it, i could not reliably test interactions with the selectbox.